client/config/default.json

```diff
@@ -3,5 +3,8 @@
   "hostname": "http://127.0.0.1:3000",
   "identifier": "TEST",
   "sendFunctionCode": false,
-  "useTimestamp": true
+  "useTimestamp": true,
+  "EAS_Exclude_Events": [],
+  "EAS_Include_FIPS": [],
+  "EAS_sendType": true
```

Comment:
My preference for this would be within an object, so something like:

```
"EAS": {
"excludeEvents": [],
"includeFIPS": [],
"sendType": true
}
```

Could use some documentation too as the config names aren't too intuitive :)

Reply:
For sure, I just made a commit to fix this, as well as updating jsame to 0.2.2 to fix a bug 👍 
I Updated the config to *below* and hopefully slightly more readable, But open to suggestions lol
```JSON
"EAS": {
    "excludeEvents": [],
    "includeFIPS": [],
    "addressAddType": true
  }
```

I cant comment the json :| we could do a wiki edit at [Client-Installation](https://github.com/pagermon/pagermon/wiki/Client-Installation#configuring-pagermon-client).
If so, we can use this ¯\_(ツ)_/¯ 

**excludeEvents:** Allows a list of [Events](https://github.com/MaxwellDPS/jsame#event-codes) to exclude ie `["RWT","RMT","SVA"]`
**includeFIPS:** Allows you to filter on a list of FIPS to alert on ie `["031109", "031000"]`
**addressAddType:** Will append the event code to the address so `KOAX-WXR` would become KOAX-WXR-W for `ZCZC-WXR-TOR-031109+0015-3650000-KOAX/NWS -` **true** or **false**

Reply:
Yeah was thinking in the README or wiki, either or.

Reply:
I dont think I can edit the wiki, so I added a client section to the README in [44d9aca90488c39e4b5e9897ef933f110328c782](https://github.com/pagermon/pagermon/pull/435/commits/44d9aca90488c39e4b5e9897ef933f110328c782)